Assets/BossRoom/Scripts/Client/Game/Action/FXProjectileTargetedActionFX.cs

```diff
@@ -100,14 +95,6 @@ private void PlayHitReact()
             }
         }
 
-        /// <summary>
-        /// Do we even have a target? (If false, it means player clicked on nothing, and we're rendering a "missed" fake bolt.)
-        /// </summary>
-        private bool HasTarget()
-        {
-            return Data.TargetIds != null && Data.TargetIds.Length > 0;
-        }
-
         private NetworkObject GetTarget()
```

Comment:
Shouldn't this match the Action's implementation? As in, if these two are matching, we won't see the target react, correct? I feel that it's weird that on the End() for the FX, we have the target trigger their hit react animation when it's a "missed" bolt.

Reply:
> As in, if these two are matching, we won't see the target react, correct?

Sorry, I can't figure out which two things you're referring to?

(If we have a target, it's never a missed bolt, so I was trying to simplify things.)

Reply:
Right. So this PR fixes the friendly fire damage, but it's still firing the hit react animation on an ally, which is what I feel can be removed. What I suggested was to align both the Action and FX GetTarget() methods so that the FX is also verifying that it's not an ally, so that you can't possibly trigger a hit react animation on your target (since target would be null on an ally).

Reply:
Oh, gotcha. Revised to address this